app/components/dashboard/DcSunburstChart.vue

```
@@ -0,0 +1,598 @@
+<script lang="ts" setup>
+import * as dc from 'dc';
+import * as d3 from 'd3';
+import { onMounted, ref, watch, defineProps, defineExpose, defineEmits, defineModel, onUnmounted, computed } from "vue";
```

ESLint

[error] 'computed' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)